WebCompiler/WebCompiler.csproj

```diff
@@ -20,6 +20,7 @@
     <PackageReference Include="Microsoft.ClearScript.V8.Native.linux-arm64" Version="7.2.0" />
     <PackageReference Include="Microsoft.ClearScript.V8.Native.osx-x64" Version="7.2.0" />
     <PackageReference Include="Microsoft.ClearScript.V8.Native.osx-arm64" Version="7.2.0" />
+    <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="6.0.0" />
```

Comment:
What a neat package, didn't even know about it! Thanks for using it and not reinventing the wheel (as I would have probably tried).

Reply:
Yes, it's quite neat isn't it. I wasn't personally aware of it until it was pointed out by one of our team members (@HennyH) here on a project we're using webcompiler on.

**Off-Topic**
I was trialling out excubo.webcompiler for our sass compilation in conjunction with buildbundleminifier for ourminify/bundle workflow for a few of projects currently working on but am currently favouring switched back to npm and gulp tasks as performance/integration-workflow appears to be better - in particular for hot-reload work.

Will revisit this as time allows and see what is missing in excubo.webcompiler (and buildbundleminifier - as the deficiencies might not be in your webcompiler packaging) compared to the npm + gulp tasks as I really like what you have done here with the .NET implementation.

Mentioning this as I note that in one of the issues you mention that you aren't using your own library in your other projects - curious to see what other limitations you may have found and whether we should consider having an open issue(s) listing these limitations for other contributors potentially to resolve?

Reply:
My use cases didn't require frequent re-compilation of scss, in fact, I barely used it. I used it for one bigger thing (material design, one-time compilation of scss for the basic theme), and used only css for the rest.

The choice of right tooling is very much individual/project-based. As long as a tool is useful to you and doesn't cause you many headaches, just use it until an even better tool comes along ;-)